tajo-algebra/src/main/java/org/apache/tajo/algebra/CreateTable.java

```diff
@@ -419,9 +420,7 @@ public Object clone() throws CloneNotSupportedException {
       hash.quantity = quantity;
       if (specifiers != null) {
         hash.specifiers = new ArrayList<>();
-        for (PartitionSpecifier specifier : specifiers) {
-          hash.specifiers.add(specifier);
-        }
+        hash.specifiers.addAll(specifiers.stream().collect(Collectors.toList()));
```

Comment:
@eminency Thanks for your comment. I agree with your opinion.
I'll rewrite it overall correlated with a simple shallow copy.